pkg/dsdk/remote_provider.go

```diff
@@ -229,3 +229,27 @@ func (e *RemoteProvider) Reload(ro *RemoteProviderReloadRequest) (*RemoteProvide
 	RegisterRemoteProviderEndpoints(resp)
 	return resp, nil, nil
 }
+
+type RemoteProviderOperationsUpdateRequest struct {
+	Ctxt        context.Context `json:"-"`
+	OperationId string          `json:"-"`
+	Action      string          `json:"action"` //available options are 'clear' and 'abort'
+}
+
+func (e *RemoteProvider) CreateOperation(ao *RemoteProviderOperationsUpdateRequest) (*RemoteProvider, *ApiErrorResponse, error) {
+
+	gro := &greq.RequestOptions{JSON: ao}
+	rs, apierr, err := GetConn(ao.Ctxt).Put(ao.Ctxt, _path.Join(e.Path, "operations", ao.OperationId), gro)
+	if apierr != nil {
+		return nil, apierr, err
+	}
+	if err != nil {
+		return nil, nil, err
+	}
+	resp := &RemoteProvider{}
+	if err = FillStruct(rs.Data, resp); err != nil {
```

Comment:
I'm not sure that the response from this endpoint will map to the RemoteProvider struct.  I tried it out in the api explorer and it looks like:
```
path:/remote_providers/fdee6a39-4ec0-4df1-9726-2ef7d4a25865/operations/4217c101-bd30-464b-8183-c96af72187f8
uuid:4217c101-bd30-464b-8183-c96af72187f8
op_state:aborted
op_type:remote_snapshot
remote_provider_uuid:fdee6a39-4ec0-4df1-9726-2ef7d4a25865
app_instance_uuid:9899b948-f808-4641-996e-1f27b4d697cb
percent_done:0
references:{} 2 items
  snapshotting:/app_instances/9899b948-f808-4641-996e-1f27b4d697cb
  snapshot_app_instance_path:/app_instances/9899b948-f808-4641-996e-1f27b4d697cb/snapshots/1580323155.547784887
total_tasks_done:0
total_tasks_issued:0
```
Seems like we might want to add a RemoteOperation struct, which could then possibly replace the map[string]interface{} of operations in the the RemoteProvider type as well?

Reply:
otoh, maybe the fields of the operation struct would vary depending on the op_type and that's why it's been left as a `map[string]interface{}`.  So maybe we should hold off on introducing a new type since we don't know  those but I do think it would be good to return some kind of information about the new state of the operation rather than the RemoteProvider struct.

Reply:
We did define a struct for these with the snapshot progress PR so we _could_ bring it over but my assumption was that we'd use maps in the interim

Reply:
Sounds good, I'm ok with returning a map from here.

Reply:
Created a RemoteOperation struct and added all values from the response.